src/foam/parsers/HTMLLexer.js

```diff
@@ -29,6 +29,23 @@ foam.ENUM({
 
 foam.CLASS({
   package: 'foam.parsers',
+  name: 'Attribute',
```

Comment:
I have placed `Attributes`, `Embed` and `Tag` under `foam.parsers.html` in the next change